### ISSUE TITLE
Support for other Transformers?

### ISSUE THREAD

**Issue by simonlevine** [gauravkdesai/MIDS-W210-Medical_Insurance_Payment_Assistant#7]
Hey! nice project. Does your implementation easily support alternative models? We just finished a very similar project more focused on the encoder side. We instantiate a pre-trained "Longformer" to accommodate the large discharge summaries but use a simple linear classifier to fine-tune. We also classify against the top 50 labels. This allows for basically no truncation, though training times are increased. Feel free to [check it out!](https://github.com/simonlevine/clinical-longformer) Perhaps we can discuss mutual improvements, as our encoder seems to outperform ClinicalBERT by a significant margin. You also may want to run with Biomed-ROBERTA-base, as this seems to outperform ClinicalBERT as well.
